apps/i18n/common/en_us.json

```diff
@@ -1011,8 +1010,6 @@
   "print": "Print",
   "privacyPolicy": "Privacy Policy",
   "projectGalleryCard": "Project Gallery",
-  "projectGalleryCardDescription": "Share your own creations or find inspiration from creations made by students all around the world.",
-  "projectGalleryCardAction": "See the gallery",
```

Comment:
Are we not using these projectGalleryCard strings anywhere? Or are there duplicates somewhere else in the file?

Reply:
We already have this exact key/value pair. My concern is that if someone in the future makes changes to this string, they might not see those changes reflected if the i18n is using the other version of the same key. Consolidating these key/value pairs while they are the same won't affect any display currently (since whichever version of the key is served, the value is the same) and protects future confusion around the keys.